src/services/validate.ts

```diff
@@ -187,7 +187,7 @@ export default class ChartValidatorService extends BaseService {
     // Find any instance of "lookup" that starts with an opening parenthesis and ignore any white spaces between opening
     // and the word itself
     // eslint-disable-next-line no-useless-escape
-    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
+    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*.*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
```

Comment:
Expecting at the word `lookup` with and space after would prevent false positives on expressions like:
```
...
{{ include "my.secrets.lookup" (dict "namespace" .Release.Namespace "secret" "my-controller-secret" "key" "ssl-cert.key" "defaultValue" $cert.Key) }}
...
```

and still capturing it on 
```
{{- $secretData := (lookup "v1" "Secret" .namespace .secret).data  -}}
```

```suggestion
    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*.*lookup\s"', `"${this.toValidate}"`], {shell: true, encoding: "utf8"});
```

Reply:
resolved, also made lookup a capture group.